apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -1073,9 +1073,9 @@ export class BookingsService_2024_08_13 {
 
       bookingLocation = getDefaultConferencingAppLocation(
         existingBookingHost?.metadata,
-        enrichedUser[0].credentials
+        enrichedUser[0].credentials ?? []
```

Comment:
Like i mentioned in the [comment](https://github.com/calcom/cal.com/pull/21180#discussion_r2141997169) - it should be `enrichedUser?.[0]?.credentials ?? [];`